android/src/main/java/com/vydia/RNUploader/UploadWorker.kt

```diff
@@ -14,41 +14,28 @@ import androidx.work.CoroutineWorker
 import androidx.work.ForegroundInfo
 import androidx.work.WorkerParameters
 import com.google.gson.Gson
-import io.ktor.client.*
-import io.ktor.client.engine.cio.*
-import io.ktor.client.plugins.*
-import io.ktor.client.request.*
-import io.ktor.client.statement.*
-import io.ktor.util.cio.*
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
 import kotlinx.coroutines.sync.Semaphore
 import kotlinx.coroutines.withContext
+import okhttp3.OkHttpClient
+import okhttp3.Response
 import java.io.File
 import java.util.concurrent.TimeUnit
 
 // All workers will start `doWork` immediately but only 1 request is active at a time.
-const val MAX_CONCURRENCY = 1
-
-// Throttling interval of progress reports
-const val PROGRESS_INTERVAL = 500 // milliseconds
+private const val MAX_CONCURRENCY = 1
 
 // Retry delay
-val RETRY_DELAY = TimeUnit.SECONDS.toMillis(10L)
-
-// Max total time for a single request to complete
-// This is 24hrs so plenty of time for large uploads
-// Worst case is the time maxes out and the upload gets restarted.
-val REQUEST_TIMEOUT_MILLIS = TimeUnit.HOURS.toMillis(24L)
```

Comment:
we no longer need to declare request timeout. In okhttp, it's default to unlimited, which is probably ok for now.

Reply:
I'm not sure if this is relevant at all in this case, but I've run into issues in the past where an unlimited timeout caused a bunch of threads to hang because of a network/dns issues.

Reply:
hmm good point, maybe I should set this to 24hrs